radix-engine/src/system/system_modules/costing/costing_module.rs

```diff
@@ -138,6 +158,16 @@ impl CostingModule {
                 .entry(key)
                 .or_default()
                 .add_assign(cost_units);
+
+            // Add an entry for the more detailed execution cost
+            cost_breakdown.detailed_execution_cost_breakdown.push((
```

Comment:
Agreed, seems like this would make sense. Will investigate making this into its own setting (I hope that it doesn't become a breaking change to transactions).